scripts/generators/patternfly-4-component/templates/component/component.tsx.hbs

```diff
@@ -7,10 +7,10 @@ interface {{ componentName }}Props {
   className?: string;
```

Comment:
Add some basic prop descriptions? Would be helpful to show their structure + our docs look for them.

Reply:
Added!